R/format_value.R

```diff
@@ -387,3 +387,28 @@ format_value <- function(x, format = NULL, output = c("ascii", "html"), na_str =
     txt
   }
 }
+
+setClassUnion("FormatSpec", c("NULL", "character", "function", "list"))
+setClassUnion("characterOrNULL", c("NULL", "character"))
+setClass("fmt_config",
+         slots = c(format = "FormatSpec", format_na_str = "characterOrNULL", align = "characterOrNULL"))
```

Comment:
There's no reason to use characterOrNULL here, befcause format_na_str will have a default string value.

Reply:
now standards are all `NULL` to be set in `rlistings`: https://github.com/insightsengineering/formatters/pull/179